queries.sql

```diff
@@ -8,3 +9,38 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered IS true;
 SELECT * from animals WHERE name NOT IN ('Gabumon');
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Day 2 */
+BEGIN; -- start transaction
+UPDATE animals SET species = 'unspecified';
+SELECT species from animals;
+ROLLBACK;
+SELECT species from animals;
+
+BEGIN; -- start transaction
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS null;
+SELECT species from animals;
+COMMIT;
+SELECT species from animals;
+
+BEGIN; -- start transaction
+DELETE FROM animals;
+SELECT COUNT(*) FROM ANIMALS;
+ROLLBACK;
+SELECT COUNT(*) FROM ANIMALS;
+
+BEGIN; -- start transaction
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM ANIMALS WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) AS average_weight from ANIMALS;
+SELECT SUM(escape_attempts), neutered FROM animals GROUP BY neutered;
```

Comment:
- Hey, in here you want to check and get the maximum of who escapes the most, neutered or not neutered animals.

Therefore you should either use the `MAX` or `AVG` instead of the sum to execute your query.